layouts/shortcodes/modular-resources.html

```diff
@@ -0,0 +1,12 @@
+<p>Additionally, you can <a href="/extend/modular-resources/configure/#add-a-module-from-the-viam-registry">use {{.Get "type" }} models from the Viam registry</a>:</p>
+<div id="searchbox"></div>
```

Comment:
We use our own Instantsearch search here, yes? So the user is searching our docs Instantsearch DB of scraped module data from the registry, which is up to 24h out of date currently. This is perfect and just what we want.

I just wanted to check, because if this is instead somehow using the registry search (like with Add a Component in the App), that search currently has strong limitations on its utility in exactly this kind of usage: "Oh I'll search for other `motor` resources since I'm on the motor page and looking for other motors" --> 0 results until https://viam.atlassian.net/browse/APP-2757.

Reply:
It is using our own typesense database but it's not being updated every 24h just yet - currently I have manually run it. I think the automation of that will be easy though and in the meantime I can run it every week or so.